mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1630,6 +1630,48 @@ func TestAccContainerCluster_withShieldedNodes(t *testing.T) {
 	})
 }
 
+func TestAccContainerCluster_withAutopilot(t *testing.T) {
+	t.Parallel()
+
+	containerNetName := fmt.Sprintf("tf-test-container-net-%s", randString(t, 10))
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config:             testAccContainerCluster_withAutopilot(containerNetName, clusterName, true),
+				ExpectNonEmptyPlan: true,
```

Comment:
Why will the plan be non-empty? For a single-phase test that's generally a sign we've got a permadiff somewhere. If this is an exceptional case where that isn't true, can context be commented?

Reply:
Got too wrapped up in checking existing tests and overlooked this. This should absolutely be removed and addressed appropriately